debian/beamium.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=beamium
+After=network.target
+
+[Service]
+EnvironmentFile=-/etc/default/beamium
+ExecStart=/bin/bash -c "if [ \"x$START_ON_INSTALL\" = \"xtrue\" -o \"x$START_ON_INSTALL\" = \"xyes\" -o \"x$START_ON_INSTALL\" = \"x0\" ] ; then /usr/bin/beamium --config=/etc/beamium/config.yaml; else /bin/echo \"beamium not configured to start, please edit /etc/default/beamium to enable\"; fi"
```

Comment:
This make the program appear in journald logs as bash.

You can get a better behavior by using `ExecPreStart` directive :
```
ExecStartPre=/bin/bash -c "if [ \"x$START_ON_INSTALL\" = \"xtrue\" -o \"x$START_ON_INSTALL\" = \"xyes\" -o \"x$START_ON_INSTALL\" = \"x0\" ] ; then exit 0; else /bin/echo \"beamium not configured to start, please edit /etc/default/beamium to enable\"; fi"
ExecStart=/usr/bin/beamium --config=/etc/beamium/config.yaml
```

That way you still have your check on start_on_install and have the correct program in journald

Reply:
I will check that this week-end and make the necessary changes.